### PR TITLE
Add documentation about running a binary on target

### PR DESCRIPTION
**Pull request by rpannek**

### CODE REVIEW

File: docs/chapters/sdk/run-binary-on-target.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Deploying and running binary on target
```
Comment (GordanM) [inline docs/chapters/sdk/run-binary-on-target.rst]: ... a binary executable

File: docs/chapters/sdk/run-binary-on-target.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+.. note:: The `Raspberry Pi documentation`_ describes how to find a new device on the network. Finding the IP address for our device will work in the same manner.
+
+.. _`Raspberry Pi documentation`: https://www.raspberrypi.org/documentation/remote-access/ip-address.md
+
+The default user and at the same time password on the PELUX image is `root`. Use `scp` to copy your binary to the target.
+
+.. code-block:: bash
+
+    scp build/template-service root@192.168.1.2:
```
Comment (GordanM) [inline docs/chapters/sdk/run-binary-on-target.rst]: Is `192.168.1.2` the default static IP or where does it come from?

File: docs/chapters/sdk/run-binary-on-target.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
+Deploying and running binary on target
+======================================
+
+Once the binary has been cross compiled it needs to be installed on the target. To do so the easiest way is to use SSH. For that to work you need to know the IP address of your target.
+
+.. note:: The `Raspberry Pi documentation`_ describes how to find a new device on the network. Finding the IP address for our device will work in the same manner.
+
+.. _`Raspberry Pi documentation`: https://www.raspberrypi.org/documentation/remote-access/ip-address.md
+
+The default user and at the same time password on the PELUX image is `root`. Use `scp` to copy your binary to the target.
```
Comment (GordanM) [inline docs/chapters/sdk/run-binary-on-target.rst]: PELUX image default user and password is `root`. 

Question: Is this true? Isn't it just root and no password?

File: docs/chapters/sdk/run-binary-on-target.rst [hunk 4]
```diff
@@ -0,0 +1,21 @@
+Deploying and running binary on target
+======================================
+
+Once the binary has been cross compiled it needs to be installed on the target. To do so the easiest way is to use SSH. For that to work you need to know the IP address of your target.
```
Comment (zeenix) [inline docs/chapters/sdk/run-binary-on-target.rst]: "," after compiled.

File: docs/chapters/sdk/run-binary-on-target.rst (hunk 2 above)
Comment (rpannek) [reply docs/chapters/sdk/run-binary-on-target.rst]: It's just a example IP address. But perhaps I shouldn't use it and instead write something like `ssh root@<ip address>`.

**Review by zeenix — APPROVED** [review]
Looks good otherwise.

**Comment by rpannek** [comment]
updated

**Review by GordanM — APPROVED** [review]

**Pull request merged**